src/azul/deployment.py

```diff
@@ -91,18 +91,15 @@ def es_endpoint(self) -> Optional[Netloc]:
 
```

Comment:
Since the property `es_endpoint` is now just a proxy for `config.es_endpoint` shouldn't it just be removed?